ipn/ipnlocal/local.go

```diff
@@ -2120,6 +2120,7 @@ func (b *LocalBackend) checkSSHPrefsLocked(p *ipn.Prefs) error {
 		if !envknob.UseWIPCode() {
 			return errors.New("The Tailscale SSH server is disabled on macOS tailscaled by default. To try, set env TAILSCALE_USE_WIP_CODE=1")
 		}
+	case "freebsd": {}
```

Comment:
No need for `{}`, Go switch statements can be empty and don't fall through by default.